wicket-core/src/main/java/org/apache/wicket/ajax/AbstractAjaxTimerBehavior.java

```diff
@@ -52,9 +52,22 @@
 
 	/**
 	 * Construct.
-	 * 
+	 *
+	 * @param updateInterval
+	 *            {@link org.apache.wicket.util.time.Duration} between AJAX callbacks
+	 *
+	 * @deprecated Since Wicket 9 this class is obsolete and no more used. It will be removed in Wicket 10. Use {@link java.time.Duration} instead
```

Comment:
s/this class/this constructor/

Reply:
You should use `{@link AbstractAjaxTimerBehavior(Duration)}` instead, i.e. point to the new constructor.

Reply:
Done!